### PR TITLE
update slo audit event info

### PR DESCRIPTION
**Pull request by roxanne-moslehi**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/service_management/service_level_objectives/_index.md [hunk 1]
```diff
@@ -211,21 +213,35 @@ Each event includes the following SLO configuration information:
...
 Three types of SLO audit events appear in the Event Explorer:
 
-1. `SLO Created` events show all four pieces of SLO configuration information at creation time.
-2. `SLO Modified` events show a what configuration information changed during a modification
-3. `SLO Deleted` events show all four pieces of configuration information the SLO had right before it was deleted
+- `SLO Created` events show the SLO configuration information at creation time
+- `SLO Modified` events show what configuration information changed during a modification
+- `SLO Deleted` events show the configuration information the SLO had before it was deleted
+
+**SLO status correction audit events:**
```
Comment (buraizu) [inline content/en/service_management/service_level_objectives/_index.md:220]:
```suggestion
### Status correction audit events
```

File: content/en/service_management/service_level_objectives/_index.md [hunk 2]
```diff
@@ -211,21 +213,35 @@ Each event includes the following SLO configuration information:
...
+
+Each event includes the following SLO status correction configuration information:
 
-To get a full list of all SLO audit events, enter the search query `tags:audit,slo` in the Event Explorer. To view the list of audit events for a specific SLO, enter `tags:audit,slo_id:<SLO ID>` with the ID of the desired SLO.
+- SLO Name
+- Status correction start and end times and timezone
```
Comment (buraizu) [inline content/en/service_management/service_level_objectives/_index.md:225]:
```suggestion
- Status correction start and end times with timezone
```

File: content/en/service_management/service_level_objectives/_index.md [hunk 3]
```diff
@@ -211,21 +213,35 @@ Each event includes the following SLO configuration information:
...
+- Status correction category
```
Comment (buraizu) [inline content/en/service_management/service_level_objectives/_index.md:226]:
```suggestion
#### Status correction audit event categories
```

File: content/en/service_management/service_level_objectives/_index.md [hunk 4]
```diff
@@ -211,21 +213,35 @@ Each event includes the following SLO configuration information:
 
 Three types of SLO audit events appear in the Event Explorer:
```
Comment (buraizu) [inline content/en/service_management/service_level_objectives/_index.md:214]:
```suggestion
#### Audit event categories

Three types of SLO audit events appear in the Event Explorer:
```

Comment (buraizu) [reply content/en/service_management/service_level_objectives/_index.md:214]: Suggesting this change for consistency with the format of the status correction audit events section

File: content/en/service_management/service_level_objectives/_index.md [hunk 5]
```diff
@@ -211,21 +213,35 @@ Each event includes the following SLO configuration information:
...
 
 {{< img src="service_management/service_level_objectives/slo-audit-events.png" alt="SLO audit events" >}}
 
-For example, if you wish to be notified when a specific SLO's configuration is modified, set an Event Monitor to track the text `[SLO Modified]` over the tags `audit,slo_id:<SLO ID>`.
+You can also use the "Audit History" tab in the SLO details to view all audit events for an individual SLO:
+
+{{< img src="service_management/service_level_objectives/slo_audit_history_tab.png" alt="SLO list page displays the Edit Tag dropdown for bulk tag editing" >}}
 
-{{< img src="service_management/service_level_objectives/slo-event-monitor.png" alt="SLO event monitor" >}}
+With [Event Monitors][28], you can set up notifications to track SLO audit events. For example, if you wish to be notified when a specific SLO's configuration is modified, set an Event Monitor to track the text `[SLO Modified]` over the tags `audit,slo_id:<SLO ID>`.
```
Comment (buraizu) [inline content/en/service_management/service_level_objectives/_index.md:244]:
```suggestion
With [Event Monitors][28], you can set up notifications to track SLO audit events. For example, if you want to be notified when a specific SLO's configuration is modified, set an Event Monitor to track the text `[SLO Modified]` over the tags `audit,slo_id:<SLO ID>`. Replace `<SLO ID>` with the SLO's ID.
```

File: content/en/service_management/service_level_objectives/_index.md [hunk 6]
```diff
@@ -211,21 +213,35 @@ Each event includes the following SLO configuration information:
...
+To get a full list of all SLO audit events, enter the search query `tags:(audit AND slo)` in the Event Explorer. To view the list of audit events for a specific SLO, enter `tags:audit,slo_id:<SLO ID>` with the ID of the desired SLO. You can also query the Event Explorer programmatically using the [Datadog Events API][19].
 
 **Note:** If you don't see events appear in the UI, be sure to set the time frame of the Event Explorer to a longer period, for example, the past 7 days.
 
 {{< img src="service_management/service_level_objectives/slo-audit-events.png" alt="SLO audit events" >}}
 
-For example, if you wish to be notified when a specific SLO's configuration is modified, set an Event Monitor to track the text `[SLO Modified]` over the tags `audit,slo_id:<SLO ID>`.
+You can also use the "Audit History" tab in the SLO details to view all audit events for an individual SLO:
+
+{{< img src="service_management/service_level_objectives/slo_audit_history_tab.png" alt="SLO list page displays the Edit Tag dropdown for bulk tag editing" >}}
```
Comment (buraizu) [inline content/en/service_management/service_level_objectives/_index.md:242]:
```suggestion
{{< img src="service_management/service_level_objectives/slo_audit_history_tab.png" alt="SLO audit history tab" >}}
```

Comment (buraizu) [reply content/en/service_management/service_level_objectives/_index.md:242]: Feel free to make any additional edits to better describe this image

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR, just requesting some minor edits, and also suggesting to add some headers for organization.

File: content/en/service_management/service_level_objectives/_index.md (hunk 3 above)
Comment (roxanne-moslehi) [reply content/en/service_management/service_level_objectives/_index.md:226]: This isn't a separate section, it's part of the list of the pieces of info in the event

**Review by buraizu — APPROVED** [review]
@roxanne-moslehi thank you for applying those changes. Since the level-2 header on line `201` has been updated, I reverted my previous suggestion for the level-3 header on line `205`. With that done, this LGTM and approving.

**Pull request merged**